tools/codegen/api/python.py

```diff
@@ -173,6 +173,53 @@
 #      }
 #
```

Comment:
(not in this PR) we probably should update the big check of comments above to reflect the new status - it's no longer purely to generate the py->c++ binding code, it's also generating python interface. Maybe add a '[Notes] python binding codegen' to the original section and a simple '[Notes] python interface codegen' to talk about special handling for interface (if any).